f5/bigip/tm/sys/software/__init__.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+#
+# Copyright 2015 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""BIG-IP® System Software module
+
+REST URI
+    ``http://localhost/mgmt/tm/sys/software``
+
+GUI Path
+    ``System``
+
+REST Kind
+    ``tm:sys:software:*``
+"""
+
+from f5.bigip.resource import LazyAttributeMixin
+from f5.bigip.resource import PathElement
+from f5.bigip.resource import ToDictMixin
+from f5.bigip.tm.sys.software.update import Update
+
+
+class Software(PathElement, LazyAttributeMixin, ToDictMixin):
```

Comment:
Ok, I will fix the mixins. So the problem with the OrganizingCollection is that it inherits ResourceBase which itself provides an update() method.

So when you use this API in practice like this

tm.sys.software.update.load()

you get an AttributeError because update is a function. :-/